pimd/pim_str.h

```diff
@@ -37,6 +41,32 @@
 #define pim_inet4_dump prefix_mcast_inet4_dump
 #define pim_str_sg_set prefix_sg2str
 
+static inline void pim_addr_copy(struct PIM_ADDR *dest, struct PIM_ADDR *source)
```

Comment:
NAK. Honestly, this should just use struct assignment.

There is absolutely nothing wrong with code like
```
struct in_addr a, b;
a = b;
```

Reply:
This change is done because we are going to add separate apis for v4 and v6. Please refer #10194. We are trying to use pim_str.h for pimv4 compilation and pim6_str.h for pimv6 compilation.

Reply:
Fixed.